packages/sundialsml/sundialsml.5.8.0p0/opam

```diff
@@ -0,0 +1,77 @@
+opam-version: "2.0"
+maintainer: "tim@tbrk.org"
+authors: [
+    "Timothy Bourke <tim@tbrk.org>"
+    "Jun Inoue <Jun.Lambda@gmail.com>"
+    "Marc Pouzet <Marc.Pouzet@ens.fr>"
+]
+homepage: "http://inria-parkas.github.io/sundialsml/"
+bug-reports: "https://github.com/inria-parkas/sundialsml/issues"
+dev-repo: "git+https://github.com/inria-parkas/sundialsml"
+doc: "http://inria-parkas.github.io/sundialsml/"
+tags: [
+    "numerical"
+    "simulation"
+    "mathematics"
+    "science"
+]
+license: "BSD-3-Clause"
+build: [
+    ["./configure" "--stubdir=%{stublibs}%/"
+		   "--libdir=%{lib}%/"
+		   "--docdir=%{doc}%/sundialsml/"
+		   "--os-distribution=%{os-distribution}%"]
```

Comment:
I tested this with macports and worked fine

Reply:
Great! Thanks for the confirmation @mseri.
I've just resubmitted an update for the parallel build problems. It also includes an attempted fix for the arm64 by compiling with `-fPIC` as suggested (at far, far right) by the error messages.
I'll try to look into the multicore build when I have to upgrade to Sundials 6.0.0, which will apparently be quite soon.